### PR TITLE
New Nav Redirects

### PR DESCRIPTION
**Pull request by softwaresale**
Added logic for redirect logic in the pages. See commit descriptions for further details

### CODE REVIEW

File: src/pages/user/profile.tsx [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
+    // TODO handle this more gracefully...
+    console.error(error)
+  }
+
+  if (!user) {
+    console.error('No user for this account exists!!')
+  }
+
+  // If the profile info is not set, then go to /user/info
+  if (!!user?.profile_info) {
```
Comment (Jsbbvk) [inline src/pages/user/profile.tsx]: Isn't `profile_info` a boolean already? And isn't this checking if `profile_info===true` then redirect when it's supposed to be `profile_info===false && redirect()`

Comment (Jsbbvk) [reply src/pages/user/profile.tsx]: also, what's the difference between `user/info` and `user/profile`?

Comment (softwaresale) [reply src/pages/user/profile.tsx]: Good point on the logic lol.

Also, /profile is the dashboard and /info is the form I think. Maybe we should merge them?

Comment (Jsbbvk) [reply src/pages/user/profile.tsx]: having `/profile` for the dashboard sounds a bit misleading imo. how about renaming `profile` to `dashboard`? and then using `profile` or `info` for the user profile form?

**Review by Jsbbvk — APPROVED** [review]

**Review by aryansw — APPROVED** [review]

**Pull request merged**